keepassxc-browser/background/keepass.js

```diff
@@ -691,7 +691,8 @@ keepass.deleteKey = function(hash) {
 keepass.getCryptoKey = function() {
     let dbkey = null;
     let dbid = null;
-    if (!(keepass.databaseHash in keepass.keyRing)) {
+
+    if (!Object.hasOwn(keepass.keyRing, keepass.databaseHash)) {
```

Comment:
The original `in` operator seems more readable, is there a specific reason for this change?

Reply:
In this case we could also keep the previous one for readability. There's no actual reason why we should use the new one.

Reply:
Keep `in` then